code/game/gamemodes/changeling/changeling.dm

```diff
@@ -1,152 +1,154 @@
-	for(var/slot in GLOB.slots)
-		if(istype(user.vars[slot], GLOB.slot2type[slot]) && !(chosen_prof.exists_list[slot])) //remove unnecessary flesh items
-			qdel(user.vars[slot])
-			continue
-
-		if((user.vars[slot] && !istype(user.vars[slot], GLOB.slot2type[slot])) || !(chosen_prof.exists_list[slot]))
-			continue
-
-		var/obj/item/C
-		var/equip = 0
-		if(!user.vars[slot])
-			var/thetype = GLOB.slot2type[slot]
-			equip = 1
-			C = new thetype(user)
-
-		else if(istype(user.vars[slot], GLOB.slot2type[slot]))
-			C = user.vars[slot]
-
-		C.appearance = chosen_prof.appearance_list[slot]
-		C.name = chosen_prof.name_list[slot]
-		C.flags_cover = chosen_prof.flags_cover_list[slot]
-		C.item_color = chosen_prof.item_color_list[slot]
-		C.item_state = chosen_prof.item_state_list[slot]
-		if(equip)
-			user.equip_to_slot_or_del(C, GLOB.slot2slot[slot])
-
-	user.regenerate_icons()
-
-
-/datum/game_mode/changeling/generate_credit_text()
-	var/list/round_credits = list()
-	var/len_before_addition
-
-	round_credits += "<center><h1>The Slippery Changelings:</h1>"
-	len_before_addition = round_credits.len
-	for(var/datum/mind/M in changelings)
-		var/datum/antagonist/changeling/cling = M.has_antag_datum(/datum/antagonist/changeling)
-		if(cling)
-			round_credits += "<center><h2>[cling.changelingID] in the body of [M.name]</h2>"
-	if(len_before_addition == round_credits.len)
-		round_credits += list("<center><h2>Uh oh, we lost track of the shape shifters!</h2>", "<center><h2>Nobody move!</h2>")
-	round_credits += "<br>"
-
-	round_credits += ..()
-	return round_credits
+GLOBAL_LIST_INIT(possible_changeling_IDs, list("Alpha","Beta","Gamma","Delta","Epsilon","Zeta","Eta","Theta","Iota","Kappa","Lambda","Mu","Nu","Xi","Omicron","Pi","Rho","Sigma","Tau","Upsilon","Phi","Chi","Psi","Omega"))
+GLOBAL_LIST_INIT(slots, list("head", "wear_mask", "back", "wear_suit", "w_uniform", "shoes", "belt", "gloves", "glasses", "ears", "wear_id", "s_store"))
+GLOBAL_LIST_INIT(slot2slot, list("head" = SLOT_HEAD, "wear_mask" = SLOT_WEAR_MASK, "neck" = SLOT_NECK, "back" = SLOT_BACK, "wear_suit" = SLOT_WEAR_SUIT, "w_uniform" = SLOT_W_UNIFORM, "shoes" = SLOT_SHOES, "belt" = SLOT_BELT, "gloves" = SLOT_GLOVES, "glasses" = SLOT_GLASSES, "ears" = SLOT_EARS, "wear_id" = SLOT_WEAR_ID, "s_store" = SLOT_S_STORE))
+GLOBAL_LIST_INIT(slot2type, list("head" = /obj/item/clothing/head/changeling, "wear_mask" = /obj/item/clothing/mask/changeling, "back" = /obj/item/changeling, "wear_suit" = /obj/item/clothing/suit/changeling, "w_uniform" = /obj/item/clothing/under/changeling, "shoes" = /obj/item/clothing/shoes/changeling, "belt" = /obj/item/changeling, "gloves" = /obj/item/clothing/gloves/changeling, "glasses" = /obj/item/clothing/glasses/changeling, "ears" = /obj/item/changeling, "wear_id" = /obj/item/changeling, "s_store" = /obj/item/changeling))
+
+/datum/game_mode	// FULP: We need changelings stored in game_mode so we can find them with hunters!
```

Comment:
we're austation, not fulpstation

Reply:
Most of those comments were on the main code, I've edited the comments so they remove that FULP: tag since I am wrapping it around austation begin and end anyway.